src/content/content.ts

```diff
@@ -57,6 +85,7 @@ const content = {
   components: {
     headerComponent,
     footerComponent,
+    personalisationComponent,
```

Comment:
Was this just for testing?

Reply:
Just that it it in "mainLayout" rather than a page specific variable.  "mainLayout" sounds like things that will be on all pages.

Reply:
Never mind, I misinterpreted the structure.